docs/js-core-1/week-1/lesson.md

```diff
@@ -528,6 +530,14 @@ Using what you've just read in the Style Guide.
 - Script: a file that contains a program
 - Terminal: another word for "console"
 
+## PD (Agile development)
+
+**Session objective:** This session provides an overview of the agile software development methodology.
+
+[Agile development slides](https://docs.google.com/presentation/d/1SN_NIXeKi4-WWWVA_ffxKyc4ZIe5Tem2zcfoYA4RtB4/edit#slide=id.g1adb8e69ab9_0_5)
+
+Notes for the facilitator: Remember to adapt/update the content in line with your cohort’s needs and arrange timings as needed.
```

Comment:
```suggestion

```
Please put this in the instructor notes https://syllabus.codeyourfuture.io/js-core-1/week-1/instructors